package.json

```diff
@@ -55,7 +55,7 @@
     "knex": "^2.4.2",
     "md5-file": "^5.0.0",
     "pg": "^8.10.0",
-    "pg-boss": "^8.1.1",
+    "pg-boss": "^9.0.3",
```

Comment:
any breaking changes to be aware of here?

Reply:
Yes, but nothing that would affect us:
https://github.com/timgit/pg-boss/releases/tag/9.0.0